lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -99,6 +117,7 @@ def find(id, opts = {})
         encode.output = []
         encode.created_at, encode.updated_at = get_times encode.id
         encode.input = build_input encode
+        encode.input.duration ||= @fixed_duration if @fixed_duration.present?
```

Comment:
`@fixed_duration` won't be set when calling `ActiveEncode::Base.find`.  You could read from this from `temp_input_metadata` here, but how about replacing the incomplete `input_metadata` instead of creating `temp_input_metadata`?  I think that might make things simpler and shouldn't lose any information.

Reply:
The issue with overwriting `input_metadata` is that it breaks the file association on Avalon's end. When we overwrite the incomplete `input_metadata` file, the URL of the copy gets inserted into the metadata. This allows the file to transcode but the outputs do not properly generate their metadata and they don't get associated with the MediaObject being created.
![improperly_formatted_output](https://user-images.githubusercontent.com/68433277/228896883-18f5109f-4e9e-42ba-b52b-baa49134cdae.png)

You can see that the label is `null` and the ID is incomplete. Ultimately, the media player does not load when viewing the item page, as if there were nothing associated with the Media Object. This occurs whether you use the complete `input_metadata` to rebuild `new_encode.input` or if you just assign the calculated duration to the original `new_encode`.

It may be possible to rewrite things to work with overwriting the file but it seemed simplest to sidestep that issue.